src/ncp/ncp_base_radio.cpp

```diff
@@ -123,6 +123,10 @@ void NcpBase::LinkRawTransmitDone(otRadioFrame *aFrame, otRadioFrame *aAckFrame,
 
         SuccessOrExit(mEncoder.BeginFrame(header, SPINEL_CMD_PROP_VALUE_IS, SPINEL_PROP_LAST_STATUS));
         SuccessOrExit(mEncoder.WriteUintPacked(ThreadErrorToSpinelStatus(aError)));
+
+        // Transmit frame header
+        SuccessOrExit(mEncoder.WriteDataWithLen(aFrame->mPsdu, static_cast<Mac::TxFrame *>(aFrame)->GetHeaderLength()));
```

Comment:
How about move line 128 to line 152, then the spinel frame will be compatible with the original spinel frame format.